cashflows/flows.py

```diff
@@ -1,10 +1,12 @@
-import warnings
-import numpy as np
 import matplotlib.pyplot as plt
+import numpy as np
+import warnings
```

Comment:
import warnings first, check pep-8 for order of imports

Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.